st2common/st2common/exceptions/db.py

```diff
@@ -16,6 +16,10 @@
 from st2common.exceptions import StackStormBaseException
 
 
+class StackStormDBKeyNotFoundError(StackStormBaseException):
```

Comment:
`DataStoreKeyNotFoundError` that @punkrokk mentioned fit to be part of `exceptions/keyvalue.py`.
Same as `ApiKeyNotFoundError` is part `exceptions/auth.py`: https://github.com/StackStorm/st2/blob/8f507d6c8d9483c8371e386fe2b7998596856fd7/st2common/st2common/exceptions/auth.py#L55-L56
which is then mentioned in the `router.py`:
https://github.com/StackStorm/st2/blob/8f507d6c8d9483c8371e386fe2b7998596856fd7/st2common/st2common/router.py#L318-L335

Does that look more like a pattern now?

Reply:
Oh. I see! I apologize.